lib/Service.php

```diff
@@ -189,6 +189,27 @@ function write($rootElementName, $value, $contextUri = null) {
 
     }
 
+    /**
+     * Maps the given plain-old-php-object (popo) to the given rootElementName within the given namespace.
```

Comment:
I agree.. was just gonna say that another benefit of using the term VO means that we can fully spell it out (as opposed to PlainOldPhpObject ;))